LibTest/io/RawDatagramSocket/writeEventsEnable_A01_t02.dart

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2022, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// @dart = 2.9
+
+/// @assertion bool writeEventsEnabled
+/// Set or get, if the [RawDatagramSocket] should listen for
+/// [RawSocketEvent.write] events. Default is true. This is a one-shot listener,
+/// and writeEventsEnabled must be set to true again to receive another write
+/// event.
+///
+/// @description Checks that if writeEventsEnabled is false, then this event is
+/// not listened.
```

Comment:
Perhaps?:
```suggestion
/// ignored.
```
(It's a little bit difficult to understand what it means for an event to be 'not listened'.)